test/transform_tests.erl

```diff
@@ -17,9 +15,14 @@
 -reflect_type([ mybool/0, myterm1/0, myterm2/0, my_int/0, my_byte/0
               , list_of_bools/0, non_empty_list_of_bools/0, mylist0/0, strings/0
               , foo_atom/0, foobarbaz/0, mytuple/0, mytuple_any/0, mytuple_empty/0
-              , simple/1, stupid_list/1, mymap/0, remote_types/0
+              , simple/1, stupid_list/1, mymap/0, remote_types/0, ipv4_address/0
+              , uri/0
               ]).
 
+-typerefl_verify({uri/0, ?MODULE, verify_uri}).
+-typerefl_from_string({ipv4_address/0, inet, parse_ipv4_address}).
+-typerefl_pretty_print({ipv4_address/0, inet, ntoa}).
```

Comment:
`ipv6` too ?

Reply:
It's just a test. Do you think it makes sense to add ipv4 and ipv6 addresses to `typerefl` as standard types?

Reply:
I was hoping for a ipv6 demonstration 
But ip parsing seem to be such a common requirement, it can never be too deep down the stack